.gitignore

```diff
@@ -2,3 +2,4 @@
 .packages
 .pub
 pubspec.lock
+.vscode
```

Comment:
Should be in ~/.gitignore-global IMHO

https://dart.dev/guides/libraries/private-files

Reply:
Ah, thanks!